SonarRunner.Shim/IRoslynV1SarifFixer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SonarQube.Common;
+
+namespace SonarRunner.Shim
+{
+    public interface IRoslynV1SarifFixer
+    {
+        bool FixRoslynV1SarifFile(string sarifFilePath, ILogger logger);
```

Comment:
It's normal to put the detailed doc comment on the interface method rather than the implementing class method.
As far as naming goes, it's not necessary to include "RoslynV1Sarif" in both the interface name and method name.